cloudant-client/src/test/java/com/cloudant/tests/HttpTest.java

```diff
@@ -186,6 +189,23 @@ public void testCookieAuthWithoutRetry() throws IOException {
         }
     }
 
+    @Test
+    public void testCookieAuthWithPath() throws Exception {
+        MockWebServer mockWebServer = new MockWebServer();
+        mockWebServer.enqueue(new MockResponse().setResponseCode(200).setBody("{\"ok\":true," +
```

Comment:
I think we have a pre-canned OK cookie response in `MockWebServerResources.OK_COOKIE`